libpromises/addr_lib.c

```diff
@@ -566,6 +566,53 @@ bool IsInterfaceAddress(const Item *ip_addresses, const char *adr)
     return false;
 }
 
+char *TrimWhitespace(char *s)
+{
+    if ( NULL_OR_EMPTY(s) )
+    {
+        return NULL;
+    }
+
+    // Leading whitespace:
+    while ( isspace(s[0]) )
```

Comment:
In general, we don't use whitespace around the expression unless needed.

It's OK when it enhances readability, for example with complex expressions or for alignment purposes.